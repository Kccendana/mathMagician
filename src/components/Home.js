function Home() {
  return (
    <div className="home-wrapper">
      <h1>Welcome to MathMagician</h1>
      <p className="home-content">Explore the realm of possibilities by engaging in calculations and drawing inspiration from our uniquely generated quotes, tailored exclusively for you.</p>
    </div>
  );
}

export default Home;
