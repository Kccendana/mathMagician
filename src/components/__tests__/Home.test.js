import React from "react";
import { render } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';
import Home from '../Home';


describe('Home component', () => {
    it('should match the snapshot', () => {
        const { container } = render(<Home />);
        expect(container).toMatchSnapshot();
    });
})