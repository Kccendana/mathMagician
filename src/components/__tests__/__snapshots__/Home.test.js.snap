// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match the snapshot 1`] = `
<div>
  <div
    class="home-wrapper"
  >
    <h1>
      Welcome to MathMagician
    </h1>
    <p
      class="home-content"
    >
      Explore the realm of possibilities by engaging in calculations and drawing inspiration from our uniquely generated quotes, tailored exclusively for you.
    </p>
  </div>
</div>
`;
